components/Footer.js

```diff
@@ -7,80 +7,124 @@ class Footer extends Component {
   render() {
     return (
       <StyledFooter>
-        <div className="newsletter-container">
-          <p className="newsletter-cta"> Get the Latest TJI Updates</p>
-          <p className="newsletter-subheading">Subscribe to our newsletter to get the latest updates:</p>
-          <MailChimpForm />
-        </div>
-
-        <div className="social-links">
-          <a
-            href="https://www.facebook.com/TXJusticeInitiative"
-            target="_blank"
-            title="Visit Texas Justice Initiative on Facebook"
-          >
-            <img src={require('../images/tji-fb-icon.svg')} alt="TJI Facebook" />
-          </a>
-          <a href="https://twitter.com/JusticeTexas" target="_blank" title="Visit Texas Justice Initiative on Twitter">
-            <img src={require('../images/tji-twitter-icon.svg')} alt="TJI Facebook" />
-          </a>
-        </div>
-
-        <div id="logo">
-          <Link href="/">
-            <img src={require('../images/tji-logo.svg')} alt="TJI Logo" />
-          </Link>
+        <div className="section-container">
+          <div className="section section__copyright">
```

Comment:
I wonder if we should use a more specific class name than section. I don't have any major issue with this, I just wonder if we might run into global styling issues at some point using this class name. Perhaps footer-section?